backend/cli.py

```diff
@@ -19,45 +18,48 @@ def cli():
     pass
 
 
+def download_file_from_s3(s3, url: ParseResult, replace=False) -> str:
+    if not s3:
+        raise ValueError("S3 client is not available")
+
+    file_name = url.path.lstrip("/")
+    logger.info("File name: %s", file_name)
+    object_information = s3.head_object(Bucket=url.netloc, Key=file_name)
+
+    if object_information["ResponseMetadata"]["HTTPStatusCode"] != 200:
+        raise ValueError(
+            f"GeoPackage file {file_name} not found in S3 bucket {url.netloc}"
+        )
+
+    logger.info("Downloading GerryDB view. Got response:\n%s", object_information)
+
+    # Download to settings.VOLUME_PATH
+    path = os.path.join(settings.VOLUME_PATH, file_name)
+
+    if os.path.exists(path) and not replace:
+        logger.info("File already exists. Skipping download.")
+    else:
+        logger.info("Downloading file...")
+        s3.download_file(url.netloc, file_name, path)
+
+    return path
+
+
 @cli.command("import-gerrydb-view")
 @click.option("--layer", "-n", help="layer of the view", required=True)
 @click.option("--gpkg", "-g", help="Path or URL to GeoPackage file", required=True)
 @click.option("--replace", "-f", help="Replace the file if it exists", is_flag=True)
 @click.option("--rm", "-r", help="Delete file after loading to postgres", is_flag=True)
 def import_gerrydb_view(layer: str, gpkg: str, replace: bool, rm: bool):
```

Comment:
(pp) can we add a comment here to clarify that import here means downloading to the volume but not doing any processing on the geopackage?